src/Dropdown/Dropdown.tsx

```diff
@@ -48,46 +46,52 @@ export interface DropdownProps extends BlockProps {
   useTriggerComponentWidth?: boolean
   onOutsideClick?: (...rest: any) => void
   test?: string
+  makeCSS?: (position: Positioning, targetDimensions: Dimensions, parentDimensions: Dimensions) => string
 }
 
 export const Dropdown: React.FC<DropdownProps> = ({
   className,
   triggerComponent,
   children,
   open,
-  horizontal = 'auto',
-  vertical = 'auto',
+  horizontal = Horizontal.AUTO,
+  vertical = Vertical.AUTO,
   breakDistance = 20,
   useTriggerComponentWidth,
   onOutsideClick,
   test = 'dropdown',
+  makeCSS = makeDefaultCSS,
```

Comment:
I am not a fan of coupling the component to this. I also do not understand why its necessary as we can easily overwrite css. If there really is a need for an ability to alter the positioning I would suggest an offset prop.